### DISCUSSION TITLE
Low power charging mode

### DISCUSSION THREAD

**Discussion by qamar-yaunos** [springfall2008/batpred#764] [General]
Hi,

Can someone please explain to me what the purpose of "low power charging mode" is?

I'm assuming this is in relation to minimising clipping during summer months? If so, what charge rate does predbat set the battery to during early morning and peak production hours?

**Comment by 7Van737** [comment]
I used this feature when I was on Eco 7 tariff and charged my battery overnight during the 7 hour off peak period.    My battery is located outside and during the below freezing nights my battery was getting down to around 5 degrees C.   

Using this mode meant the battery reduced the charge rate to make use of the whole off peak charge window - rather than charge at the maximum rate (which would have taken just a few hours of the window.    The extended charge period ensured my battery temperature was "acceptable" over the whole charge window and not just the beginning of it.

Predbat calculates the charge rate so as to make use of most of the available charge window.   This feature is only really worth using for tariffs which have fixed lengthy off peak periods - like Eco, Go etc.   I'm now on Agile and don't use the feature.

**Reply by qyaunos1** [reply]
Thanks for that reply. Is anyone aware of any features within predbat that help to minimise clipping?

Issue I'm going to have in summer is that I've got a east West array 10.44kwp split evenly across both aspects.

5kwh hybrid gen 3 inverter and a 9.5kwh battery.

I'm aware that I'm going to have clipping, but one way to mitigate this is by ensuring that the battery is as empty as possible by the time the peak generation time comes. Therefore I want to be able to have 5kwh shared between the house and exports and the remainder to go in to the battery at up to 3.68kwh an hour.

**Reply by 7Van737** [reply]
Predbat should handle this for you - maximising the "free" PV.  Think what is critical is accurate modelling of your installation on Solcat - so that forecasted output is as near to actual as possible.   

Clipping won't be as issue for me so have not given it too much thought.   Suggest you also post this question on more 'popular' forums  like https://community.givenergy.cloud/ and Facebook GivEnergy/GivTCP groups so others who may have similar setups to you can advise.

**Comment by springfall2008** [comment]
If you set your inverter rate correctly then Predbat will model the clipping and should try to improve the situation for you. That doesn't always mean keeping the battery low as there is a balance between clipping and running out of charge later

**Reply by qyaunos1** [reply]
Thanks for the reply Trefor. Appreciate it.

Quick question please if you don't mind - Im finally ready to switch off "read only" mode.

Is there anything to do in the givenergy app with my charge schedule to avoid it conflicting with what predbat is trying to do?

If not, how does predbat override the settings in the givenergy app?

**Comment by springfall2008** [comment]
> Thanks for the reply Trefor. Appreciate it.
> 
> Quick question please if you don't mind - Im finally ready to switch off "read only" mode.
> 
> Is there anything to do in the givenergy app with my charge schedule to avoid it conflicting with what predbat is trying to do?
> 
> If not, how does predbat override the settings in the givenergy app?

If you enabled any automations in the cloud settings then its best to turn them off first otherwise its good to go
